test/harry/main/org/apache/cassandra/harry/cql/SelectHelper.java

```diff
@@ -233,7 +233,7 @@ private static Where.Inequalities toInequalities(Relations.RelationKind kind)
     private static CompiledStatement toCompiled(Select select)
     {
         // Select does not add ';' by default, but CompiledStatement expects this
-        String cql = select.toCQL().replace("\n", " ") + ";";
+        String cql = select.toCQL(CQLFormatter.None.instance) + ';';
```

Comment:
for context Alex had a patch to add this behavior into `Select` and there was some back and forth about that... so he dropped the logic and used this replace... since this patch adds formatters we can do the same without needing the replace now.